dream_layer_backend/test_run_registry.py

```
@@ -0,0 +1,430 @@
+        # Verify all runs are gone
+        runs = temp_registry.list_runs()
+        assert len(runs) == 0
+        
+        # Verify individual runs are gone
+        for run_id in run_ids:
+            assert temp_registry.get_run(run_id) is None
+    
+    def test_thread_safety(self, temp_registry):
+        """Test that operations are thread-safe"""
+        import threading
+        import time
```

Ruff (0.12.2)

`time` imported but unused

Remove unused import: `time`

(F401)